Samples/All/BumbleRumble/Common/NetworkManager.h

```diff
@@ -51,13 +51,19 @@ class NetworkManager : public Manager
     inline NetworkManagerState State() { return m_state; }
     inline bool IsConnected() const { return m_state == NetworkManagerState::NetworkConnected; }
 
+	inline bool IsCognitiveServicesEnabled() { return m_enableCognitiveServices; }
+	void SetCognitiveServicesEnabled(bool bEnabled);
+
 private:
     bool InternalConnectToNetwork(const char* networkId, Party::PartyNetworkDescriptor& descriptor);
     void CreateLocalUser();
+	void CreateLocalChatControl();
     std::string DisplayNameFromChatControl(Party::PartyChatControl* control);
     void UpdateTTSProfile();
+	void SetTextChatTranslationOptions(bool bTranslateToLocalLanguage);
+	void SetVoiceChatTranscriptionOptions(bool bTranscribeSelf, bool bTranscribeOtherChatControlsWithMatchingLanguages, bool bTranscribeOtherChatControlsWithNonMatchingLanguages, bool bDisableHypothesisPhrases, bool bTranslateToLocalLanguage);

```

Comment:
strange formatting/indenting with the parameters.  if it is because it is all on one line, better to break up the lines for readability.

Reply:
I don't know why it is displaying like this. I will look at it again.